src/main/riddl/Members/memberTypeDefinitions.riddl

```diff
@@ -0,0 +1,58 @@
+//type MemberType is any of {Student, FacultyStaff, Vendor, Sponsor, General, Alumni, Parent, Fan}
```

Comment:
Asking because I'm not sure what the convention is for us:

Is MemberType not going to be used at all in the future? I only leave these commented out as I'm not sure what the intention is for such types. Are these commented out because they are no longer used and are there any plans to be implemented? This question is also applicable to any other commented out riddl documentation in the repository.

Reply:
MemberType will likely not be used. Instead the Organization will be able to define a set of Tags that can be used on a memberType field of a Member.  That came from a discussion where we can't know the full set of enumerators for MemberType since we don't want to limit applicability just to schools. It should work for any kind of organization (like Improving!).

Reply:
Sounds good. I'm removing MemberType from members.riddl in #35 then.